src/gis/features.md

```diff
@@ -24,9 +24,16 @@ If you are using a vector layer for background layer:
 -   All vector layers not intended to be used as survey layers, have to be set as read-only. To make a vector layer read-only in QGIS, from the main menu select Project \> Properties. Within the window, select Data Source tab and select the Read-only option for the layers you do not want to be used as survey layer in Input.
 
 ## Project extent
-In Input app, there is an option to zoom to the project extent. If not set, Input zooms to all visible layers. This is not particularly convenient when you have a layer with a large/global extent (e.g. Open Street Map).
+In <MobileAppName />, there is an option to [zoom to the project extent](../field/input_ui/#zoom-to-project-browse-features-map-themes-settings).
 
-To set the project extent: from **Project** > **Properties**, select **QGIS Server** (not the most obvious location!). Under **WMS capabilities** select the option for **Advertised extent** and either enter the coordinate extent of your project bounding box or use the canvas extent.
+To set the project extent, navigate to **Project** > **Properties**. Select **View Settings** and check the **Set Project Full Extent** option. Here, either enter the coordinate extent of your project bounding box or use the map canvas extent.
+
+![qgis project extent](./qgis-project-extent.png)
+
+If the project extent is not set, <MobileAppName /> will use **Advertised extent** of WMS. You can find this option in **QGIS server** tab in the project properties.
+![qgis WMS extent](./qgis-project-advertised-wms-extent.png)
```

Comment:
The usage of advertised extent in QGIS server tab is kind of legacy from times when "view settings" did not exist yet, not sure if it is worth mentioning (I would probably at least omit the screenshot, so that people are not visually drawn to this option)